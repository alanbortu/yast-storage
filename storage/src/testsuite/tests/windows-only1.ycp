// testedfiles: helper1b.ycp
{
    include "./lib/setup-system.ycp";

    setup_system("windows-only");

    include "./lib/helper1a.ycp";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", true);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", false);
    ProductFeatures::SetStringFeature("partitioning", "root_max_size", "20 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "15 GB");

    include "./lib/helper1b.ycp";
}
