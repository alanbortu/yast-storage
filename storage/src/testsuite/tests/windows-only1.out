Proposal:
Shrink Windows partition /dev/sda1 to 33.21 GB [destructive]
Create extended partition /dev/sda2 (41.32 GB)
Create swap partition /dev/sda5 (517.72 MB)
Create root partition /dev/sda6 (16.50 GB) with ext4
Create partition /dev/sda7 (24.31 GB) for /home with ext4
